<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFreeMenuWidget</class>
 <widget class="QWidget" name="CFreeMenuWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>86</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="pFrameMenuLogo">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="pLabelMenuLogoImg">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>54</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLabel" name="pLabelMenuLogoText">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>108</width>
        <height>12</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
     <zorder>pLabelMenuLogoImg</zorder>
     <zorder>pLabelMenuLogoText</zorder>
     <zorder>pLabelMenuLogoImg</zorder>
     <zorder>pLabelMenuLogoText</zorder>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="pMenuToolWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="43,44">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="pMenuCloseWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="pMenuWidget" native="true"/>
         </item>
         <item>
          <spacer name="pMenuCloseSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="pCloseWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="pCloseToolBtnLayout">
              <property name="spacing">
               <number>0</number>
              </property>
             </layout>
            </item>
            <item>
             <spacer name="pCloseWidgetSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>91</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
        <zorder>pMenuWidget</zorder>
        <zorder>pMenuCloseSpacer</zorder>
        <zorder>pCloseWidget</zorder>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="pToolWidget" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="margin">
          <number>0</number>
         </property>
         <property name="spacing">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="pToolWidgetLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
